build/deploy/generate-deploy-examples.sh

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env bash
+
+
+SCRIPT_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
+MANIFEST_FILE=../../deploy/examples/multus-validation.yaml
+SED=${SCRIPT_ROOT}/../../build/sed-in-place
+
+
+
+run_gen() {
+  if [[ ${SKIP_GEN} = true ]]; then
+    echo "SKIP_GEN is set, skipping manifest  generation"
+    exit
+  fi
+      OUT_FILE="${SCRIPT_ROOT}/${MANIFEST_FILE}"
+      TMP_FILE="$(mktemp)"
+
+ START_MARKER='# THE BELOW HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+ END_MARKER='# THE ABOVE HELP TEXT IS AUTO-GENERATED BY `make docs`$'
+
+
+
+	  go run ${SCRIPT_ROOT}/../../cmd/rook/*.go multus validation run --help | awk '{print "#" $0}' > ${TMP_FILE}
+
+	${SED} -e "/$START_MARKER/,/$END_MARKER/{ /$START_MARKER/{ p ; r $TMP_FILE
+                }; /$END_MARKER/p; d; }" -i.orig ${OUT_FILE}
```

Comment:
Is `-i.orig` going to leave a new file that git attempts to try to track? What happens if this flag is removed?

Reply:
@BlaineEXE  wrote:
> Is `-i.orig` going to leave a new file that git attempts to try to track?
that is correct: it leaves an untracked backup file

>  What happens if this flag is removed?

I just tried locally and it modifies the manifest in-place without creating an untracked backup file.

Reply:
@BlaineEXE, I added a commit to remove it ...